piksi_tools/console/settings_view.py

```diff
@@ -433,7 +417,7 @@ def _selected_setting_changed(self):
 
     def _expert_changed(self, info):
         try:
-            self.settings_display_setup(do_read_finished=False)
+            self.settings_display_setup(finish=False)
```

Comment:
finish" is less clear to me.  What does it mean? "do_read_finished" behaves as named.

Reply:
reverted